src/js/listeners/auth/login.test.js

```diff
@@ -0,0 +1,13 @@
+import login from "./login";
+describe("stores a token in browser",() => {
+
+    it("adds token", () =>{
+        expect(login.location).toequal("`./?view=profile&name=${name}")
```

Comment:
`login.location` is not a real property, and you are not expected to interact with the URL for this test.